cwf/gateway/integ_tests/test_runner.go

```diff
@@ -300,6 +303,11 @@ func (tr *TestRunner) WaitForReAuthToProcess() {
 	time.Sleep(4 * time.Second)
 }
 
```

Comment:
fancy where is this used? or is this for debugging

Reply:
Oh yes i was using this for debugging and forgot to remove it.... But it is useful in case we ever want to print timestamps during a test